packages/react-shopify-app-route-propagator/README.md

```diff
@@ -3,58 +3,23 @@
 [![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator.svg)](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator) ![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-shopify-app-route-propagator.svg)
 
-`<ShopifyRoutePropagator />` is a simple component to synchronize a Shopify embedded app's client side routing with the outer iframe host. It assumes the embedded app is either using [Shopify's App Bridge Library](https://help.shopify.com/en/api/embedded-apps/app-bridge) or [Polaris v3+ with Shopify App Bridge](https://polaris.shopify.com/components/structure/app-provider#section-initializing-the-shopify-app-bridge)
+This package contains `useRoutePropagator` hook and `<RoutePropagator />` component to synchronize a Shopify embedded app's client side routing with the outer iframe host. It assumes the embedded app is either using [Shopify's App Bridge Library](https://help.shopify.com/en/api/embedded-apps/app-bridge) or [Polaris v3+ with Shopify App Bridge](https://polaris.shopify.com/components/structure/app-provider#section-initializing-the-shopify-app-bridge)
 
-The component is quite small and can be used with any routing solution.
+The package is quite small and can be used with any routing solution.
 
 ## Installation
 
 ```bash
 $ yarn add @shopify/app-bridge @shopify/react-shopify-app-route-propagator
 ```
 
-## Prepare app instance
-
-Two typical way of getting an app instance are as follow:
-
-1. Creating the app as suggested by the [app bridge documentation](https://help.shopify.com/en/api/embedded-apps/app-bridge#set-up-your-app)
-
-```typescript
-import createApp, {getShopOrigin} from '@shopify/app-bridge';
-const app = createApp({
-  apiKey: 'API key from Shopify Partner Dashboard',
-  shopOrigin: getShopOrigin(),
-});
-```
-
-2. Getting it from [polaris-react](https://github.com/Shopify/polaris-react)'s context. Note that this is only avaliable on [polaris-react](https://github.com/Shopify/polaris-react) v3 or higher.
-
-```typescript
-import React from 'react';
-import * as PropTypes from 'prop-types';
-
-class MyApp extends React.Component {
-  // This line is very important! It tells React to attach the `polaris`
-  // object to `this.context` within your component.
-  static contextTypes = {
-    polaris: PropTypes.object,
-  };
-
-  render() {
-    ...
-    const app = this.context.polaris.appBridge;
-    const routePropagatorMarkup = app ? (
-      <ShopifyRoutePropagator location={location} app={app} />
-    ) : null;
-  }
-}
-```
-
 ## Usage
 
-The component has the following props signature
+In both `useRoutePropagator` and `<RoutePropagator />` you need two to pass in two variables with the signature:
```

Comment:
I think it might be clearer to write this as something like

```
Both the hook and component versions of this library take the same two parameters:
```